improver/nbhood/use_nbhood.py

```diff
@@ -137,6 +182,7 @@ def __init__(
                 mask is not applied. Therefore, the neighbourhood processing
                 may result in values being present in areas that were
                 originally masked.
+                This should be set to False if using collapse_weights.
```

Comment:
Maybe worth to check this in code and throw an error or warning instead of just documenting?

Reply:
Yep, done.